src/components/profile/AvatarCircle.js

```diff
@@ -99,7 +99,7 @@ export default function AvatarCircle({
     ios || !isAvatarPickerAvailable ? React.Fragment : ContextMenuButton;
 
   return (
-    <Wrapper handlePressMenuItem={handlePressMenuItem} menuConfig={items}>
+    <Wrapper menuConfig={items} onPressMenuItem={handlePressMenuItem}>
```

Comment:
What's the difference `handlePress...` vs `onPress...`?

Reply:
I just renamed this for consistency